recipes/rtmidi/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class RtMidiConan(ConanFile):
+    name = "rtmidi"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.music.mcgill.ca/~gary/rtmidi/"
+    description = "Realtime MIDI input/output"
+    topics = ("midi")
+    license = "Copyright (c) 2003-2019 Gary P. Scavone"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "pkg_config"
+    exports_sources = "patches/*"
```

Comment:
Missing `CMakeLists.txt`.
```suggestion
    exports_sources = "CMakeLists.txt", "patches/*"
```

Reply:
I added the CMakeLists.txt wrapper in my last commit. I wonder why this is needed by the way.
I had to patch rtmidi for that: should I make a PR for this patch to the upstream repository?

Reply:
It's always a sign of good cmake hygiene if you're project allows to be used as a subproject.
The patch can be upstreamed with the added motivation that it allows vendoring rtmidi or using it as a git submodule.

Reply:
Done here: https://github.com/thestk/rtmidi/pull/252

It is not so relevant for the upstream repository since it concern the install target which is not used when rtmidi is a submodule.

The project I used as example to write cmake based recipe is szip which use a `_build_subfolder` property: https://github.com/conan-io/conan-center-index/blob/master/recipes/szip/all/conanfile.py#L37-L39

Is there any reason to use this property?